web-console/future-openshift/diagnostics/diagnostics.md

```diff
@@ -0,0 +1,37 @@
+# Diagnostic Report
+
+Users can download a diagnostic report of their system and send it to Red Hat from the console.
+
+
+## Help menu
+![dropdown](img/dropdown.png)
+* The "Diagnostic Report" option is available from the help menu.
+
+## Modal
+![modal](img/modal.png)
```

Comment:
For the selection of what to include – should it first be select the nodes, then select all of the relevant diagnostics to run (like a sosreport for RHEL on each node)? +1 to liz's comment about "all nodes" could be the default selection. For the support case, I wonder if having a distinct menu item (image/dropdown.png) for opening a support case could simplify this a little? For example, if there was an alternate workflow from the initial dropdown "open a support case" then the option to "include diagnostics" with a new case. The option to include to an existing case seems really useful in this dialog – but if you remove the other options, you could perhaps simplify this section to just a checkbox of "add diagnostics to an existing support case" then if checked it could expose the text area to enter the case number?